cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -759,7 +772,7 @@ runTxBuildRaw
         proposals
         mCurrentTreasuryValueAndDonation
 
-    first TxCmdTxBodyError $ createAndValidateTransactionBody sbe txBodyContent
+    first TxCmdTxBodyError $ createTransactionBody sbe txBodyContent
```

Comment:
I imagine this is correct, but I don't know why we are not validating here anymore